### PR TITLE
Warning fix

### PR DESCRIPTION
**Pull request by TomyLobo**
some other minor changes included as well

### CODE REVIEW

**Comment by ZNickq** [comment]
Squash the commits! Other than that, looks fine!

File: src/main/java/org/spout/vanilla/generator/normal/decorator/MushroomDecorator.java [hunk 1]
```diff
@@ -79,11 +79,13 @@ public void populate(Chunk chunk, Random random) {
...
 
-    private void generateMushroom(Chunk c, Random ra, int cx, int cy, int cz) {
+    @SuppressWarnings("unused")
+	private void generateMushroom(Chunk c, Random ra, int cx, int cy, int cz) {
         c.setBlockMaterial(cx, cy, cz, getRandomMushroomType(ra, false), (short) 0, true, c.getWorld());
     }
 
-    private void generateHugeMushroom(World world, BlockMaterial type, Random ra, int x, int y, int z) {
+    @SuppressWarnings("unused")
+	private void generateHugeMushroom(World world, BlockMaterial type, Random ra, int x, int y, int z) {
```
Comment (zml2008) [inline src/main/java/org/spout/vanilla/generator/normal/decorator/MushroomDecorator.java]: This formatting doesn't look right. Either this line or the lines around it should be corrected.

**Comment by TomyLobo** [comment]
the first 2 commits are separate self-contained changes. why should i squash them?

**Comment by TomyLobo** [comment]
ok i fixed the space thing. added a commit to change the whole file to tabs too

**Pull request merged**